packages/lesswrong/lib/collections/electionVotes/helpers.ts

```diff
@@ -204,3 +204,11 @@ export const submissionCommentsToFormState = (submissionComments?: SubmissionCom
     note: submissionComments?.rawFormValues.note ?? "",
   }
 }
+
+/** Approximately the time the election was accounced: https://forum.effectivealtruism.org/posts/x2KfyNe8oPR4dqGkf/ea-forum-plans-for-giving-season-2023 */
```

Comment:
This was the last thing required from `eaGivingSeason.ts`, so I just moved it here